### ISSUE TITLE
Add Diva.js to a CDN

### ISSUE THREAD

**Issue by ahankinson** [DDMAL/diva.js#164]
Add Diva to the cdnjs libraries.

http://cdnjs.com

**Comment by magoni** [comment]
https://github.com/cdnjs/cdnjs/pull/5665

**Comment by magoni** [comment]
```
https://cdnjs.cloudflare.com/ajax/libs/diva.js/4.0.0/js/diva.min.js
https://cdnjs.cloudflare.com/ajax/libs/diva.js/4.0.0/css/diva.min.css
```
:beers: :cake:

**Comment by magoni** [comment]
I do have to fix the image links in the CSS to point to the CDN before everything's usable, but it's online! :horse:

**Comment by ahankinson** [comment]
:+1:

**Comment by magoni** [comment]
Image issues will be resolved when 4.1 is integrated into master, so moving it to the 4.1 milestone as a reminder to add CDN links to the documentation.
